content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/endpoint.md

```diff
@@ -0,0 +1,138 @@
+---
+title: zenoss.endpoint() function
+description: >
+  The `zenoss.endpoint()` function sends events to Zenoss using data from input rows.
+menu:
+  influxdb_2_0_ref:
+    name: zenoss.endpoint
+    parent: Zenoss
+weight: 202
+---
+
+The `zenoss.endpoint()` function sends events to Zenoss using data from input rows.
+
+_**Function type:** Output_
+
+```js
+import "contrib/bonitoo-io/zenoss"
+
+zenoss.endpoint(
+  url: "https://example.zenoss.io:8080/zport/dmd/evconsole_router",
+  username: "example-user",
+  password: "example-password",
+  action: "EventsRouter",
+  method: "add_event",
+  type: "rpc",
+  tid: 1
+)
+```
+
+## Parameters
+
+### url
+({{< req >}})
+Zenoss events [public endpoint](https://help.zenoss.com/zsd/RM/configuring-resource-manager/enabling-access-to-browser-interfaces/creating-and-changing-public-endpoints) URL.
```

Comment:
"Zenoss router endpoint URL" is more accurate `url` parameter description I would say  now after I've read https://help.zenoss.com/dev/collection-zone-and-resource-manager-apis/anatomy-of-an-api-request#AnatomyofanAPIrequest-RouterURL (I wish I had found it before, sorry!).